pkg/controller/volume/persistentvolume/pv_controller.go

```diff
@@ -1342,13 +1421,45 @@ func (ctrl *PersistentVolumeController) provisionClaim(claim *v1.PersistentVolum
 	}
 	klog.V(4).Infof("provisionClaim[%s]: started", claimToClaimKey(claim))
 	opName := fmt.Sprintf("provision-%s[%s]", claimToClaimKey(claim), string(claim.UID))
-	startTime := time.Now()
-	ctrl.scheduleOperation(opName, func() error {
-		pluginName, err := ctrl.provisionClaimOperation(claim)
-		timeTaken := time.Since(startTime).Seconds()
-		metrics.RecordVolumeOperationMetric(pluginName, "provision", timeTaken, err)
-		return err
-	})
+	plugin, storageClass, err := ctrl.findProvisionablePlugin(claim)
```

Comment:
I don't think we need to do all of this logic to distinguish between CSI and intree plugins.  Let's have the same metric for all types

Reply:
fare enough, I was doing that to avoid any changes to existing logic, if we could afford changing the original behavior (though it has some flaw), I am happy to merge these two

Reply:
fixed